src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py

```diff
@@ -202,6 +221,19 @@ def _hardcoded_place(self):
             )
         return action
 
+    def _rl_place(self, obs: Observations, info: Dict[str, Any]):
+        place_step = self.timesteps[0] - self.place_start_step[0]
+        if place_step == 0:
+            action = DiscreteNavigationAction.MANIPULATION_MODE
+        elif self.place_done[0] == 1:
+            action = DiscreteNavigationAction.STOP
+        else:
+            action, term = self.place_agent.act(obs)
```

Comment:
Can you make this terminate? Took me some time to realize what term meant

Reply:
Renamed.